### DISCUSSION TITLE
Linux vs Windows ? Who wins?

### DISCUSSION THREAD

**Discussion by metamountain** [Comfy-Org/ComfyUI#2979] [Q&A]
Can someone tell me , if  there is a measurable advantage of using linux ? Could someone who has both installations post some figures ?
Speed and ram usage experience would be nice ! Which linux distribution is easiest to start with ?

**Comment by rethink-studios** [comment]
+1

**Comment by ltdrdata** [comment]
I prefer Ubuntu. Just easy to use.

**Comment by scottonanski** [comment]
Just go with the Windows version... The Linux install is a pain in the ass and not worth the headache.

**Reply by BrechtCorbeel** [reply]
Even with smatrix? I'm installing Ubuntu soon for my workstation to offload the GPU usage from my main computer, so I'll run as a server And connect over LAN in the browser.
